packages/frontend/src/hooks/useTrackSiteReload.ts

```diff
@@ -0,0 +1,18 @@
+import { SITE_EVENTS } from '@utils/amplitude'
+import { useEffect } from 'react'
+import useAmplitude from './useAmplitude'
+
+const useTrackSiteReload = () => {
+  const { track } = useAmplitude()
+  const win: any = typeof window !== 'undefined' ? window : undefined
+
+  useEffect(() => {
+    if (win?.performance) {
```

Comment:
if anyone from FE team is okay with this. I am good :)

Reply:
Looks good